### DISCUSSION TITLE
Instead of deactivating inputs allow user to queue up runs?

### DISCUSSION THREAD

**Discussion by voodoohop** [pollinations/pollinations#97] [Pollinations]
I imagine a kind of queue to which one can add inputs and it runs them sequentially. Would be model-independent. E.g. you can queue up runs from any models.

**Comment by gokaykucuk** [comment]
Good idea, but we have to come up with a ui component to show the queue.

**Comment by voodoohop** [comment]
Question: should it be in the frontend(pollinations.ai) or the backend(pollinator).
